tests/fv/fv_test.go

```diff
@@ -305,16 +305,6 @@ var _ = Describe("kube-controllers FV tests", func() {
 			_, err = calicoClient.IPPools().Create(context.Background(), &pool, options.SetOptions{})
 			Expect(err).NotTo(HaveOccurred())
 
-			// Claim affinity to a block.
```

Comment:
This step was unnecessary since the AssignIP call does it automatically below.

It also left the datastore in an intermediate state such that the new IPAM controller changes detected as a candidate for garbage collection.